packages/components/src/components/Divider/Divider.stories.tsx

```diff
@@ -17,12 +22,17 @@ export default meta;
 
 export const Divider: StoryObj<typeof DividerComponent> = {
     render: props => (
-        <Container>
+        <Container orientation={props.orientation}>
             <DividerComponent {...props} />
+            <DividerComponent {...props}>
+                <Text variant="tertiary">OR</Text>
```

Comment:
I suggest to add children as a prop too and keep only one divider in storybook. Otherwise both components look like a one. Or you can also create a separate story